fw/t/fuzzer.c

```diff
@@ -3,11 +3,11 @@
  *
  * Tempesta HTTP fuzzer.
  *
- * Copyright (C) 2015-2017 Tempesta Technologies, Inc.
+ * Copyright (C) 2015-2022 Tempesta Technologies, Inc.
  *
  * This program is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License,
+ * the Free Software Foundation; either version 0.2.0 of the License,
```

Comment:
No, the version of the module in line 32, not the GPL version :)

Reply:
Shame on me :(